packages/app/src/components/pages/Home/NewFilesToReview.js

```diff
@@ -3,20 +3,43 @@ import { NominationsDataContext } from '../../../utils/context/NominationsContex
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import NewNomination from './NewNomination';
 import styles from "./styles.css";
+import useSort from './useSort'
 
 const NewFilesToReview = () => {
   const [NominationsData, setNominationsData] = useContext(NominationsDataContext);
   const [showAll, setShowAll] = useState(false);
   const receivedNominations = NominationsData ? NominationsData.filter(nominations => nominations.status === "received") : []
+  const { sortedNoms, requestSort, sortConfig } = useSort()
+  const sortedNominations = sortedNoms ? sortedNoms.filter(nominations => nominations.status === "received") : []
```

Comment:
I'm confused by the separation between `receivedNominations` and `sortedNominations`. Why would we ever use `receivedNominations` when it looks like `sortedNominations` are the same thing, but sorted/sortable?

Also, please use single-quotes instead of double-quotes. I know we're mixing a bit, but single-quotes are always preferred.